### DISCUSSION TITLE
브라우저의 작동 방식

### DISCUSSION THREAD

**Discussion by giscus** [autroshot/dev-archive#57] [Comments]
# 브라우저의 작동 방식

사용자는 콘텐츠가 빠르게 로딩되고 상호 작용이 부드러운 웹 경험을 원합니다. 개발자는 이 두 가지 목표를 달성하기 위해 노력해야 합니다.

https://www.dev-archive.com/blog/%EB%B8%8C%EB%9D%BC%EC%9A%B0%EC%A0%80%EC%9D%98-%EC%9E%91%EB%8F%99-%EB%B0%A9%EC%8B%9D/

**Comment by autroshot** [comment]
테스트
